views/partials/component/sidebar.html

```diff
@@ -0,0 +1,23 @@
+<div class="o-registry-ui__component--sidebar">
+	<div class="o-registry-ui__component--sidebar--content">
+		<div class="o-registry-ui__component--sidebar-links">
+			<a class="github-link" href="{{component.url}}">GITHUB REPOSITORY</a>
+			<a class="external-link" href="{{component.url}}#readme">README</a>
+			<!-- TODO: implement css/js conditions. -->
+			<a class="external-link" href="http://codedocs.webservices.ft.com/v1/jsdoc/{{component.name}}@v{{component.version}}">JSDoc</a>
+			<a class="external-link" href="http://codedocs.webservices.ft.com/v1/sassdoc/{{component.name}}@v{{component.version}}">SassDoc</a>
+		</div>
+
+		{{#ifEquals component.type 'imageset'}}
+			{{> component/quickstart/imageset}}
+		{{/ifEquals}}
+
+		{{#ifEquals component.type 'module'}}
+			{{> component/quickstart/build-service}}
+		{{/ifEquals}}
+
+		{{#ifEquals component.type null}}
+			{{> component/quickstart/build-service}}
```

Comment:
When the component `type` is `null` we can't really assume that it runs with the build service. I'd be tempted to leave this part out.